README.md

```diff
@@ -378,14 +378,31 @@ easier.
 You should not collect results intended for publication with development switches
 turned on.
 
-We also recommend that for 'real' benchmarking you turn on the
-`--hardware-reboots` and `--daemonise` switches. These ensure that the system
-will reboot before each benchmark execution, and that Krun will run in the
-background, allowing you to log out before the first reboot.
-
-You will also need to ensure that Krun is restarted once the machine has
-rebooted. The `etc/` directory contains example `/etc/rc.local` files for the
-platforms supported by Krun.
+We also recommend that benchmarking in "reboot mode". The recommended way to do
+this is via `cron(8)`. We supply a script `start_krun_from_cron` to make this
+easier.
+
+As a regular user (who has access to your experiment and the Krun code), invoke
+`crontab -e` and add and add lines similar to the following:
+
+```
+CONFIG_FILE=your_experiment.krun
+KRUN_DIR=/path/to/krun/dir
+EXPERIMENT_DIR=/path/to/dir/containing/config/file
+@reboot /usr/bin/env /bin/sh -c ${KRUN_DIR}/scripts/start_krun_from_cron
```

Comment:
The `/usr/bin/env` thing is pointless, as you give it an absolute path. Just `/bin/sh` will be OK.

I'm not very keen on specifying some things through "global" environment variables though (it's not obvious how everything works). Could we not make `start_from_krun` take those two things as parameters or ...?

Reply:
We can do it as parameters if you like.

The only reason I used the env, was so I could re-use `${KRUN_DIR}`, which admittedly is superficial.

Reply:
OK, sounds like parameters are the way to go then?